src/scripts/clipperUI/components/modeButton.tsx

```diff
@@ -35,7 +35,7 @@ class ModeButtonClass extends ComponentBase<{}, PropsForModeButton> {
 			<a className={className} role="option" aria-selected={this.props.selected}
 				id={idName} title={this.props.tooltipText ? this.props.tooltipText : ""}
 				aria-setsize={this.props["aria-setsize"]} aria-posinset={this.props["aria-posinset"]}
-				{...this.enableAriaInvoke({callback: this.buttonHandler, tabIndex: this.props.tabIndex, ariaSetName: Constants.AriaSet.modeButtonSet, ariaSetDirection: AriaNavDirection.Vertical})}>
+				{...this.enableAriaInvoke({callback: this.buttonHandler, tabIndex: this.props.tabIndex, ariaSetName: Constants.AriaSet.modeButtonSet, ariaSetDirection: AriaNavDirection.Vertical, autoSelect: false})}>
```

Comment:
We can remove all the places where we declared autoSelect=false since false is now the default value